ChainSafe/gossamer
# fix(cmd/gossamer): embed default toml config files

## Changes
Embed default toml config files as resources so that they can be loaded from binary instead of local file system.  This includes files that match pattern `chain/*/*.toml`

## Tests


- Build gossamer (from project directory) `make gossamer`
- `cd` to a different directory
- Run gossamer `path/bin/gossamer`

Notice that gossamer runs without errors:

```
2023-02-01T15:04:49-05:00 ERROR    failed to open toml configuration file: open /home/emack/chain/gssmr/config.toml: no such file or directory  toml_config.go:L29      pkg=cmd
2023-02-01T15:04:49-05:00 ERROR    failed to load toml configuration: open /home/emack/chain/gssmr/config.toml: no such file or directory       config.go:L66   pkg=cmd
2023-02-01T15:04:49-05:00 ERROR    failed to set chain configuration: open /home/emack/chain/gssmr/config.toml: no such file or directory       config.go:L109  pkg=cmd
2023-02-01T15:04:49-05:00 ERROR    failed to create node configuration: open /home/emack/chain/gssmr/config.toml: no such file or directory     main.go:L224    pkg=cmd
open /home/emack/chain/gssmr/config.toml: no such file or directory
```

## Issues
fixes: #3020 

## Primary Reviewer

@timwu20
